de.bforartists.Bforartists.json

```diff
@@ -0,0 +1,176 @@
+{
+    "id": "de.bforartists.Bforartists",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "bforartists",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=host",
```

Comment:
the rationale doesn't stick.
```
--filesystem=/media
--filesystem=/run/media
```

is sufficient for removable devices.

Reply:
I agree, and I'm fine with this change. That makes more sense than just giving full host access. Would this be a more acceptable set of permissions? `home` and `/mnt` is what the Blender Flatpak used before moving to `host`.

```json
"--filesystem=home",,
"--filesystem=/mnt",
"--filesystem=/media",
"--filesystem=/run/media"
```

Reply:
These new permissions have been set here: https://github.com/flathub/flathub/pull/3357/commits/8b949050934f4070b67e46d4fb48a43c080e93a0